amcl/CMakeLists.txt

```diff
@@ -1,141 +1,153 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(amcl)
 
-find_package(catkin REQUIRED
-        COMPONENTS
-            rosbag
-            roscpp
-            tf
-            dynamic_reconfigure
-            nav_msgs
-        )
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
```

Comment:
Are we switching to CMAKE_CXX_STANDARD like: https://github.com/ros2/rmw/pull/109